src/hera.cpp

```diff
@@ -150,15 +150,21 @@ void execute(
 
   // This should be caught during deployment time by the Sentinel.
   // TODO: validate for other conditions too?
-  heraAssert(module.getExportOrNull(Name("main")) != nullptr, "Contract entry point (\"main\") missing.");
+  heraAssert(module.getExportOrNull(Name("main")) != nullptr && module.getExportOrNull(Name("_main")) != nullptr, "Contract entry point (\"main\" or \"_main\") missing.");
```

Comment:
```module.getExportOrNull(Name("main")) != nullptr && module.getExportOrNull(Name("_main")) != nullptr```
The assertion should not require that both "main" and "_main" exports exist. Should be using OR operator instead of AND.

Reply:
yeah good call.  I had that change in my local environment but forgot to push it up.